examples/darwin-framework-tool/BUILD.gn

```diff
@@ -42,6 +42,9 @@ declare_args() {
   }
 
   enable_provisional_features = config_enable_yaml_tests
+
+  # Enable generating compiler database
+  gen_compdb = false
```

Comment:
I would expand this to something less abreviated
`darwin_generate_compile_database` or `genrate_compilation_database` or similar?